code/modules/events/event_dynamic.dm

```diff
@@ -99,6 +99,8 @@ var/list/event_last_fired = list()
 	//and start working via the constructor.
 	new picked_event
 
+	score["eventsendured"]++
```

Comment:
Technically speaking, a random event is a random event. And outside of the lottery I'm pretty sure they are all harmful in some fashion